docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -59,19 +59,30 @@ image::images/view-alert-details.png[Shows the Event Renderer button, 200]
+** *Investigation time enrichments*: Displays for all alerts regardless of the rule type that generated it. Like indicator rule enrichments, investigation time enrichments are also pre-fixed by the {es-sec} icon as a way to set them apart from general alert details. Time enrichment data from the past 30 days is generated when the {es-sec} app queries the `filebeat-*` index for the following set of fields. If these field values aren't found, investigation time enrichments aren't created. Or, if an indicator match rule uses these fields for indicator mapping, and the indicator rule enrichment data matches the  investigation time enrichment data, the investigation time enrichments are treated as duplicated data and are removed. For example, if you specify `file.hash.sha256` and `file.hash.md5` as fields that should be mapped for your indicator match rule, these fields will only display as indicator rule enrichments on the **Summary** and **Threat Intel** tabs when the rule is activated and the alert is generated. You can learn more about the investigation time enrichment query by going to the **Threat Intel** tab and clicking the **Inspect** icon next to event you want to inspect.
 
-NOTE: If an alert is linked to more than one threat, a compiled version of threat indicator data displays in the *threat summary* section. A more detailed view displays within the *Threat Intel* tab.
+- `file.hash.md5`: The MD5 hash
+- `file.hash.sha1`: The SHA1 hash
+- `file.hash.sha256`: The SHA256 hash
+- `file.pe.imphash`: Imports in a PE file
+- `file.elf.telfhash`: Imports in an ELF file
+- `file.hash.ssdeep`: The SSDEEP hash
+- `source.ip`: The IP address of the source (IPv4 or IPv6)
+- `destination.ip`*: The event's destination IP address
+- `url.full`: The full URL of the event source
+- `registry.path`: The full registry path, including the hive, key, and value
 
-* *Threat Intel*: Shows the number of matched threats and displays them individually. Threats appear in reverse chronological order, with the most recent alerts at the top. The available `threat.indicator` and `source.event` data is displayed for each threat. If the alert has not been enriched with threat data, the *Threat Intel* tab displays the message "No Threat Intel Enrichment Found" and provides a link to Threat Intel module documentation.
+NOTE: To further verify the type of threat intelligence data you're viewing, hover your mouse over the field name. Also note that if an alert is linked to more than one threat, matched field values  display within the *Summary* tab and a more detailed view can be found within the *Threat Intel* tab.
```

Comment:
maybe we can just say `hover over the field name` as that's the only way to hover on something as far as I understand. Although I think for accessibility users should be able to pop that up with the keyboard as well.